shared/login/login/container.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+import * as Creators from '../../actions/login/creators'
```

Comment:
this is really moving index.js to container like our new code. moving things into helpers and using recompose to eliminate the helper class